utilities/ldap_sync.sql

```diff
@@ -1,11 +1,138 @@
+   for this script. Examples:
+       "LDAP Server URLS" --> "ldap://192.168.1.155:389"
+       "CREATE CONNECTION "test_ldap_server" to 'ldap://192.168.1.155:389....'
+   The reasoning is to eliminate failed connections using the DNS name of the 
+   LDAP Server. Out of the box, Exasol does not know of your DNS or LDAP Servers,
+   and some forget to set the "DNS Server 1" (and 2) on the Network page. 
+   
+5) First implement the unsecured LDAP functionality, (using http prefix "ldap://" 
+   ensuring the proper results, then plan to implement the secured LDAP setup. 
+   It's much easier to implement in smaller tasks, than to try an implement the ideal setup all at once.
+   
+6) Build these scripts after you have made your changes, such as the name of the CONNECTION and which SCHEMA to use. 
+   This will ensure core functionality is implemented. To assist with verifying LDAP connections
+   and general troubleshooting, compile "LDAP_HELPER" using the information you compiled. Specifically,
+   build your CONNECTION using the "CREATE CONNECTION" SQL command and set up the SQL to call
+   "LDAP_HELPER" with the LDAP distinguished name entry. Examples:
+       a) Build Connection:
+           create or replace connection test_ldap_server to 'ldap://192.168.1.155:389' user 'cn=admin,dc=example,dc=com' identified by 'secret';
+       b) Build the SQL to call "LDAP_HELPER":
+           select <your_schema>.LDAP_HELPER_('TEST_LDAP_SERVER', 'cn=John Doe,ou=Users,dc=example,dc=com') ;
+   
+7) On a closing note, when you first test the newly implemented functionality, this script will timeout
+   after 30 seconds, if a valid connection is not made. Should this happen, these are the
+   most likely culprits:
+   a) The LDAP I.P. address and Port are incorrect.
+   b) You configured your CONNECTION using the LDAP DNS server name and missed setting
+      the EXASolution UI entries "DNS Server 1". You can rebuild your CONNECTION using
+      the LDAP Server I.P address and retry.
+   c) There is a firewall between Exasol and the LDAP server. You can also use
+      this link to build just a connection tester:
+      https://github.com/exasol/exa-toolbox/blob/master/utilities/check_connectivity.sql
+      if you are sure the LDAP I.P. address is correct, then maybe the port is wrong.
+      Try running the "check_connectivity" SQL using a different port, such as 80.
+      Here are some common well known port numbers:
+            21: FTP Server.
+            22: SSH Server (remote login)
+            25: SMTP (mail server)
+            53: Domain Name System (Bind 9 server)
+            80: World Wide Web (HTTPD server)
+            110: POP3 mail server.
+            143: IMAP mail server.
+            443: HTTP over Transport Layer Security/Secure Sockets Layer (HTTPDS server)
+
+
+======================================
+Change Log
+======================================
+
+Version 2.2
```

Comment:
I believe we usually add info on new versions below (like in [CREATE_DB_DDL](https://community.exasol.com/t5/database-features/create-ddl-for-the-entire-database/ta-p/1417)).